addToCart/js.js

```diff
@@ -0,0 +1,37 @@
+const getEl = (el) => document.querySelector(el);
+
+/* All the elements used */
+const shopingButton = getEl('.shoping_button');
+const cartSvg = getEl('svg.shoping_cart');
+const cartHolder = getEl('.shopingcart_wrapper');
+const buttonTxt = getEl('p.button_text');
+const completedEl = getEl('.completed');
+
+/* Flag to keep track if the mouse was pressed or not */
+let mouseDown = false;
+
+/* When mouse is pressed change flag, and move the cart */
+shopingButton.addEventListener('mousedown',(e)=>{
+  mouseDown = true;
+  cartSvg.style.transform = 'translateX(-10px)'
```

Comment:
when you change css properties by js try do not forget the prefix like -webikit -moz etc..

Reply:
Done,
Sure, i will do it from now on ;)